src/pfs/pfs.proto

```diff
@@ -300,6 +300,20 @@ message CreateBranchRequest {
   bool new_commit_set = 5; // overrides the default behavior of using the same CommitSet as 'head'
 }
 
+message FindCommitsRequest {
+  Commit start = 1;
+  string file_path = 2;
+  uint32 limit = 3; // a limit of 0 means there is no upper bound on the limit.
+}
+
+message FindCommitsResponse {
+  oneof result {
```

Comment:
why do we return only one of these?

Reply:
This message is returned as a stream. If we find 5 commits, there will be 1 message per commit. On the client side, we stitch the response together into a flat struct that we can present to the user.